tools/run

```diff
@@ -46,5 +46,11 @@ if $prod; then
   command="JEKYLL_ENV=production $command"
 fi
 
+if [ -e /proc/1/cgroup ]; then
+  if grep -q docker /proc/1/cgroup; then
+    command="$command --force_polling"
```

Comment:
`--livereload` is very useful. Would you mind to include it by default too?

Reply:
I didn't delete it.

Reply:
https://github.com/cotes2020/jekyll-theme-chirpy/blob/709794923f75dcde814ddbedffb54e9519daa2fd/tools/run#L6

Reply:
My bad. `-l` is so tiny to notice 😆